plugins/catalog-react/src/components/InspectEntityDialog/components/OverviewPage.tsx

```diff
@@ -90,6 +91,12 @@ export function OverviewPage(props: { entity: AlphaEntity }) {
                 <ListItemText primary="etag" secondary={metadata.etag} />
               </ListItem>
             )}
+            <ListItem>
+              <ListItemText
+                primary="entityRef"
+                secondary={stringifyEntityRef(props.entity)}
```

Comment:
Hey this is great, while we're here coooould I perhaps ask nicely for a copy button for the value too? :D

You can use a [`CopyTextButton`](https://github.com/backstage/backstage/blob/29716aa3039ec05358e26fcedd4e437673f3dd74/packages/core-components/src/components/CopyTextButton/CopyTextButton.tsx#L80) as a [`ListItemSecondaryAction`](https://v4.mui.com/components/lists/#interactive) for example 🙏

Reply:
will give it a look later today :D was thinking about that as well.

Do you think any other fields could use this same functionality?

Reply:
Sure!

Reply:
done for etag, uid and entityRef. In my initial thoughts apiversion, entitykind and spec.type don't benefit much from having a copy button. If you feel like I should add it for those too for consistency, I'd be happy to add it there too.